python/samples/documentation_examples/planner.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft. All rights reserved.
+import asyncio
+import importlib.util
+import os
+
+from service_configurator import add_ai_service
+
+import semantic_kernel as sk
+from semantic_kernel.planners.sequential_planner import SequentialPlanner
+
+
+def get_math_plugin():
```

Comment:
why is this so complex here? why not`import_native_plugin_from_directory`?

Reply:
Why is `import_native_plugin_from_directory` so complex? We should be able to specify a different module name instead of tying it to `native_function.py`.